src/main/cpp/RobotContainer.cpp

```diff
@@ -95,13 +109,19 @@ RobotContainer::RobotContainer()
   frc::SmartDashboard::PutData("reset balls in conveyor", new ResetBallConveyor(mConveyor));
 
   // Configure the button bindings
+
   ConfigureButtonBindings();
 }
 
 void RobotContainer::ConfigureButtonBindings() {
   // Configure your button bindings here
   frc2::Trigger( [this] { return mConveyor.IsBallComing(); }).WhenActive( [this]{ mConveyor.BallIntakeIncoming(); });
   frc2::Trigger( [this] { return mConveyor.IsBallExiting(); }).WhenInactive( [this] { mConveyor.BallIntakeExiting(); });
+  frc2::Button dA {[this]{return driver.GetRawButton(1);}};
+  frc2::Button dB {[this]{return driver.GetRawButton(2);}};
```

Comment:
What has button B been added to the driver controller? We probably want it but it's not assigned to anything.